<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ATVista</class>
 <widget class="QMainWindow" name="ATVista">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATVista</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>722</width>
      <height>202</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="ATButton" name="btnSalir">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>65</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>65</height>
          </size>
         </property>
         <property name="text">
          <string>Salir</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="ATLabel" name="AtechnikLogo">
         <property name="minimumSize">
          <size>
           <width>397</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>397</width>
           <height>120</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ATButton" name="btnStartStop">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>170</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>170</height>
        </size>
       </property>
       <property name="text">
        <string>st</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ATButton</class>
   <extends>QPushButton</extends>
   <header>/usr/local/include/lib_ATGUI/at_button.h</header>
  </customwidget>
  <customwidget>
   <class>ATLabel</class>
   <extends>QLabel</extends>
   <header>/usr/local/include/lib_ATGUI/at_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
